tests/e2e/example.test.e2e.js

```diff
@@ -0,0 +1,57 @@
+// @ts-check
+import {test, expect} from '@playwright/test';
+import {login_user, save_visual, load_logged_in_context} from './utils_e2e.js';
+
+test.beforeAll(async ({browser}, workerInfo) => {
+  await login_user(browser, workerInfo, 'user2');
+});
+
+test('Load Homepage', async ({page}) => {
+  const response = await page.goto('/');
+  await expect(response?.status()).toBe(200); // Status OK
+  await expect(page).toHaveTitle(/^Gitea: Git with a cup of tea\s*$/);
```

Comment:
Probably not a good test for this string as it can be changed via configuration. Remove or try to parse the configuration.